sdk/src/main/java/io/dapr/actors/runtime/AppToDaprHttpAsyncClient.java

```diff
@@ -4,205 +4,111 @@
  */
 package io.dapr.actors.runtime;
 
-import io.dapr.client.AbstractDaprHttpClient;
-import io.dapr.exceptions.DaprException;
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import io.dapr.client.ClientRequestBuilder;
 import io.dapr.utils.Constants;
-import io.dapr.utils.ObjectSerializer;
 import okhttp3.OkHttpClient;
 import reactor.core.publisher.Mono;
 
-import java.util.HashMap;
-import java.util.Map;
-
 /**
  * Http client to call Dapr's API for actors.
  */
 //public class DaprHttpAsyncClient implements DaprAsyncClient {
-class AppToDaprHttpAsyncClient extends AbstractDaprHttpClient implements AppToDaprAsyncClient {
```

Comment:
Move all implementation to AbstractDaprHttpClient - there is no need for this to be separate.

Reply:
you mean move this to DaprHttp.java? this is very specific to Actors, probably rename it to ActorsHttpClient and move it package io.dapr.actors.client?